### DISCUSSION TITLE
Tours disappeared after update

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#1010] [General]
Hello
I juste remove the old version in order to install the brand new one. Oh surprise, all tours previousely saved are no longer here. Can I get them back,or what did I do wrong?
Thank you

**Comment by mt-from-sf-to-gh** [comment]
You can see all saved tours when

* `All People` is selected
* `All Tour Types` is selected
* All tour filters are **not** selected

![image](https://user-images.githubusercontent.com/103518804/223981679-78ae5862-8adc-4359-a582-2c4033455808.png)

**Reply by mt-from-sf-to-gh** [reply]
What have you done when you write `I juste remove the old version` ?

**Comment by wolfgang-ch** [comment]
I just discovered, that I was logged in with my 2nd accound https://github.com/mt-from-sf-to-gh

**Comment by gringojack** [comment]
I uninstalled using Total Uninstal by Martau. That program I use for all of my applications. It also is possible that I removed a little too much!

**Reply by wolfgang-ch** [reply]
It looks like that `Total Uninstall` is also deleting the tour data. 

There is no need to uninstall anything when using the MT Windows installer, it is deleting the old program files and installing the new program files, **any tour data are untouched.**

**Comment by gringojack** [comment]
Thank you Wolfgang for this advice. It is my fault. Total Uninstal works very good. I just have not to remove pieces of data at the end of removal, just when Total Uninstal ask me the question

**Comment by gringojack** [comment]
I uninstalled, installed and now no more MTB I get these messages
![2023-03-10 09_31_28-](https://user-images.githubusercontent.com/4082252/224304864-32617c62-baf5-4ec3-9c6c-872d96607faa.jpg)
What should I do???
Thank you

**Reply by wolfgang-ch** [reply]
Your tour data directory seems to be corrupt, do these steps to do a clean installation

* Uninstall MT
* Delete folder C:\Users\<YOUR NAME>\mytourbook\
* Install MT

**Comment by gringojack** [comment]
Thank you Wolfgang, it works fine
Now I have to redefine the whole program from january 1st
I do road  and Mountain  biking. How can I differenciate one from the other?
Thank you
Jacky

**Reply by wolfgang-ch** [reply]
With tour types https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/tools/tour-type
